### PR TITLE
Origami major cascade

### PR DESCRIPTION
**Pull request by andygout**
One breaking test, which we will look into next week. Possibly to do with `[next-test-sessions-lambda](https://github.com/financial-times/next-test-sessions-lambda)`.

### CODE REVIEW

File: demos/scss/demo.scss [hunk 1]
```diff
@@ -1,10 +1,16 @@
+$system-code: n-profile-ui !default;
+
 @import 'n-ui-foundations/main';
 @import '../../main.scss';
 
+html {
+	background-color: #fff1e5;
```
Comment (chee) [inline demos/scss/demo.scss]:
```suggestion
	background-color: oColorsByName('paper');
```

File: test/client/live-consent.spec.ts [hunk 2]
```diff
@@ -2,23 +2,18 @@ import * as sinon from 'sinon';
 import stubFetch from './helpers/stub-fetch';
 import initConsentForm from './helpers/init-consent-form';
 
-let afterFormSaved = () => { };
-
 describe('Live consent form', () => {
 	beforeEach(() => {
-		initConsentForm(() => { afterFormSaved(); });
+		initConsentForm(() => { () => { } });
```
Comment (chee) [inline test/client/live-consent.spec.ts]: confused by this, there is no return statement. Can't this be:

```
initConsentForm(() => {})
```

? because the inner function is never returned?

Comment (chee) [reply test/client/live-consent.spec.ts]: in fact, the `init-consent-form` function is only used here, so we could remove the callback from it altogether

File: test/client/live-consent.spec.ts [hunk 3]
```diff
@@ -2,23 +2,18 @@ import * as sinon from 'sinon';
...
-	it('shows an updated switch value when the form is saved', async done => {
+	it('shows an updated switch value when the form is saved', async () => {
 		stubFetch();
 		expect(radioButton().checked).toEqual(false);
 		radioButton().click();
-		afterFormSaved = () => {
-			expect(radioButton().checked).toEqual(true);
-			done();
-		};
+		expect(radioButton().checked).toEqual(true);
```
Comment (chee) [inline test/client/live-consent.spec.ts]: this does change the behaviour of this test quite a bit, so it might be good to include some information in the commit messages about why these decisions were made

File: test/client/live-consent.spec.ts (hunk 2 above)
Comment (andygout) [reply test/client/live-consent.spec.ts]: After this is merged and released (to unblock other repos) I am returning directly to revise this test file to create a meaningful test, which will require fixing this function.

File: test/client/live-consent.spec.ts (hunk 3 above)
Comment (andygout) [reply test/client/live-consent.spec.ts]: The test description is `it('shows an updated switch value when the form is saved'`.

While this is true, the switch value is updated before the form is saved, demonstrated by taking the expectation out of the `afterSavedForm` context.

The more meaningful (and less misleadingly described) test will be to ensure that the `√ Saved` element displays, which happens only once the form is saved.

Options are:
- Release this in its current state to unblock other repos and I will immediately work on revising this file to create a meaningful test (and release patch version).
- Work on revising this file to do as described above now, which will block other repos from being worked on in the meantime.

File: src/scss/form.scss [hunk 4]
```diff
@@ -1,132 +1,73 @@
...
-.consent-form__label {
-	@include oTypographySans(0);
-	font-weight: normal;
-}
-
-.consent-form__label-bold {
-	@include oTypographySans(2);
-	font-weight: oFontsWeight(semibold);
-}
-.consent-form__item-advisory {
```
Comment (edds) [inline src/scss/form.scss]: Why was this removed?
